cmd/apply.go

```diff
@@ -18,9 +18,11 @@ package cmd
 
 import (
 	"fmt"
+	"os"
+
 	pkgcmd "github.com/kedgeproject/kedge/pkg/cmd"
+
```

Comment:
this happened according to this https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859